packages/block-editor/src/components/justify-toolbar/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup } from '@wordpress/components';
+import {
+	justifyLeft,
+	justifyCenter,
+	justifyRight,
+	justifySpaceBetween,
+} from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+const icons = {
+	left: justifyLeft,
+	center: justifyCenter,
+	right: justifyRight,
+	'space-between': justifySpaceBetween,
+};
+
+export function JustifyToolbar( {
```

Comment:
Is the name good enough for a public API? Is it consistent with other `*Toolbar` components? Maybe it is, just making sure it's considered carefully.